src/properties/components/Properties.tsx

```diff
@@ -5,26 +5,65 @@
  * https://github.com/eclipsesource/jsonforms-editor/blob/master/LICENSE
  * ---------------------------------------------------------------------
  */
+import {
+  materialCells,
+  materialRenderers,
+} from '@jsonforms/material-renderers';
+import { JsonForms } from '@jsonforms/react';
 import { Typography } from '@material-ui/core';
+import { isEmpty, isEqual } from 'lodash';
 import React from 'react';
 
-import { FormattedJson } from '../../core/components';
-import { useSelection } from '../../core/context';
-import { toPrintableObject } from '../../core/model';
+import { useDispatch, useSelection } from '../../core/context';
+import { Actions } from '../../core/model';
+import { getRoot } from '../../core/util/clone';
+import { ExamplePropertiesService } from '../propertiesService';
 
+const propertiesService = new ExamplePropertiesService();
 export const Properties = () => {
   const [selection] = useSelection();
+  const dispatch = useDispatch();
+
```

Comment:
instead of checking all places in the code whether selection exists we could do a quick exit (after all the hook calls), e.g.
```
if(!selection) {return <NoSelection />;}
```